modules/core/src/main/scala/org/tessellation/http/routes/DebugRoutes.scala

```diff
@@ -41,6 +41,8 @@ final case class DebugRoutes[F[_]: Async](
       services.gossip.spread(ConsensusEvent(toEvent(TimeSnapshotTrigger()))) >> Ok()
     case POST -> Root / "consensus" / "trigger" / LongVar(height) =>
       services.gossip.spread(ConsensusEvent(toEvent(TipSnapshotTrigger(Height(height))))) >> Ok()
+    case POST -> Root / "consensus" / "stop" =>
```

Comment:
I was thinking about `reset`, since for `run-validator` nodes it's actually resetting to the initial state. However for the `run-genesis` nodes it doesn't do reset to the initial state, it just stops the consensus so I went with `stop` (also I was thinking about `halt`).

Reply:
Hm, genesis is also a snapshot so when you set none it's being reset, isn't it?

Reply:
Oh, it doesn't reset the global snapshot storage. Got it. but on the validator nodes global snapshots are still in the storage too, so "stop" is correct then